README.md

```diff
@@ -18,7 +18,7 @@ From left to right: WinUI, iOS and Android.
 ![](doc/images/dialog.png)
 
 ## Getting Started
-Here is how to setup the service in your WinUI or mobile apps made with Uno Platform. If you want to know how to setup this service for UWP, check out [getting started with UWP](doc/GettingStartedUWP.md).
```

Comment:
I wouldn't remove this. It still works on UWP, it's just no longer supported starting at version 2.

Reply:
We could move it so that it's less prominent though, like in a lower section of the readme.